app/src/test/java/org/mozilla/fenix/ext/ImageButtonTest.kt

```diff
@@ -0,0 +1,33 @@
+package org.mozilla.fenix.ext
+
+import kotlinx.coroutines.ObsoleteCoroutinesApi
+import org.mozilla.fenix.TestApplication
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import io.mockk.mockk
+import io.mockk.verify
+import android.widget.ImageButton
+
+@ObsoleteCoroutinesApi
+@RunWith(RobolectricTestRunner::class)
+@Config(application = TestApplication::class)
```

Comment:
fwiw, this is outside the scope of this PR but you can reduce duplication in the tests by merging the config setting into your test runner so each test looks like this:
```kotlin
@RunWith(FirefoxRobolectricTestRunner::class)
class SomeTest {
```

We do that in Fire TV: https://searchfox.org/mozilla-mobile/rev/2a57cc10d768832199bfd755d1e08fd9630baccc/firefox-tv/app/src/test/java/org/mozilla/tv/firefox/helpers/FirefoxRobolectricTestRunner.kt#26